assets/charts/control-plane/prometheus-operator-crds/Chart.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: prometheus-operator-crds
+description: A Helm chart for Prometheus Operator CRDS
```

We should follow per k8s documentation.

```suggestion
description: A Helm chart for Prometheus Operator CRDs.
```